src/main/scala/is/hail/io/plink/PlinkLoader.scala

```diff
@@ -23,17 +23,12 @@ object PlinkLoader {
 
   val plinkSchema = TStruct(("rsid", TString()))
 
-  private def parseBim(bimPath: String, hConf: Configuration, a2Reference: Boolean = true): Array[(Variant, String)] = {
+  private def parseBim(bimPath: String, hConf: Configuration, a2Reference: Boolean = true,
+    contigRecoding: Map[String, String] = Map.empty[String, String]): Array[(Variant, String)] = {
     hConf.readLines(bimPath)(_.map(_.map { line =>
       line.split("\\s+") match {
         case Array(contig, rsId, morganPos, bpPos, allele1, allele2) =>
-          val recodedContig = contig match {
-            case "23" => "X"
-            case "24" => "Y"
-            case "25" => "X"
-            case "26" => "MT"
-            case x => x
-          }
+          val recodedContig = contigRecoding.getOrElse(contig, contig)
```

Comment:
Do you want the default behavior to still do the `"23" => "X"`, etc thing? or should it now get explicitly remapped?

Reply:
It should be explicit in Scala. For `import_plink` on Python method, I added the defaults because that is really common. PLINK stores the contigs as integers, so a lot of files will be coded that way. If we change the default genome reference to 38, then we'll need to update that default value for the remapping.